h2o-core/src/main/java/water/H2O.java

```diff
@@ -298,6 +301,9 @@ public static void printHelp() {
     /** -login_conf is login configuration service file on local filesystem */
     public String login_conf = null;
 
+    /** -ssl_config path (absolute or relative) to a file containing all SSL related configurations */
+    public String ssl_config = null;
```

Comment:
oki, good point

Now, i am slightly confused what @tomkraljevic is expecting. Lets discuss that directly